src/Websites/Websites/Cmdlets/WebApps/PublishAzureWebApp.cs

```diff
@@ -72,6 +75,17 @@ public override void ExecuteCmdlet()
                 using (var s = File.OpenRead(ArchivePath))
                 {
                     HttpClient client = new HttpClient();
+                    if (Timeout != 0)
```

Comment:
comparing a parameter with the default value of its type is not an obvious way to check if use specifies it. Try this:
```suggestion
                    if (this.IsParameterBound(cmdlet => cmdlet.Timeout))
```

Reply:
Thanks for your suggestion. made it as suggested.